test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js

```diff
@@ -0,0 +1,126 @@
+import { RequestMock, RequestLogger } from 'testcafe';
```

Comment:
After figured out what's best to determine user agent, maybe try investigate https://devexpress.github.io/testcafe/documentation/guides/advanced-guides/intercept-http-requests.html#create-a-custom-request-hook to see if helps test for mobile flow. (in your next pr)

Reply:
I added a unit test.

Reply:
unit test on the `isMobile` logic in the view?

Reply:
https://github.com/okta/okta-signin-widget/pull/1333/files#diff-d7cb6251c5518c2baa815dd881270044R38

Reply:
would be ideal if we could demonstrate the user agent mock in testcafe :D